repository/src/main/resources/db/changelog/db.changelog-rdm-1240.3.xml

```diff
@@ -0,0 +1,83 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-1240-3" author="giri.benadikar@hmcts.net">
+        <insert tableName="case_field">
+            <column name="reference" value="jurisdiction"/>
```

Comment:
I believe there was a reason to use the exact notation with upper case and square brackets as reference for metadata types:
```
[JURISDICTION]
```

Reply:
This would remove the need for additional logic when mapping to the definition and it also would prevent collision with flexible case fields as [] are not valid characters for those (this is relevant for the state case field which is created as non-global).

Reply:
This comment also applies to any references below

Reply:
The additional logic when mapping metadata fields to definition was trivial compared to sanitising the name of metadata fields for retrievals (the names are in-line with the standard search API).

Reply:
Ok, then we have the risk of collision for state which means we need to add a validation rule around that in definition store to prevent anyone from using `state` as a field name.